src/components/NavBar/UkDisclaimerModal.tsx

```diff
@@ -0,0 +1,55 @@
+import { Trans } from '@lingui/macro'
+import { ButtonEmphasis, ButtonSize, ThemeButton } from 'components/Button'
+import Column from 'components/Column'
+import Modal from 'components/Modal'
+import Row from 'components/Row'
+import { useCloseModal, useModalIsOpen } from 'state/application/hooks'
+import { ApplicationModal } from 'state/application/reducer'
+import styled from 'styled-components'
+import { CloseIcon, ThemedText } from 'theme/components'
+
+const Wrapper = styled(Column)`
+  padding: 8px;
+`
+
+const ButtonContainer = styled(Column)`
+  padding: 8px 12px 4px;
+`
+
+const StyledThemeButton = styled(ThemeButton)`
+  width: 100%;
+`
+
+export function UkDisclaimerModal() {
+  const isOpen = useModalIsOpen(ApplicationModal.UK_DISCLAIMER)
+  const closeModal = useCloseModal()
+
+  return (
+    <Modal isOpen={isOpen} onDismiss={closeModal}>
+      <Wrapper gap="md">
+        <Row justify="flex-end" padding="8px 0px 4px">
+          <CloseIcon size={24} onClick={() => closeModal()} />
+        </Row>
+        <Column gap="sm">
+          <ThemedText.HeadlineLarge padding="0px 8px" fontSize="24px" lineHeight="32px" fontWeight={500}>
```

Comment:
fontWeight 500 is not in the spores color refresh, use 485 for normal or 535 for bold

Reply:
also you used a themedtext here but overwrote all the styles

Reply:
500 is the size design told me to use on the figma - and all of these styles are also based on the figma

Reply:
These design might be pre-spore, we shouldn't introduce uses of the old font-weights. Tagging in @infredible and @bryankuuu